swan-lake/development-tutorials/source-code-dependencies/package-references.md

```diff
@@ -59,9 +59,9 @@ visibility = "private"
 
 ### The `org` field
 
-The organization is a logical name used for grouping modules together under a common namespace within a repository. Building a library package with `bal build` and pushing a library package into a repository will fail without an organization name.
+The organization is a logical name used for grouping modules together under a common namespace within a repository. 
```

Comment:
why removing this sentence?

Reply:
Removed via https://github.com/ballerina-platform/ballerina-dev-website/pull/8583#discussion_r1461556684. @azinneera

Reply:
The compilation itself fails and so we don't need to say about publishing. The next sentence says that its mandatory which gives the information.